docs/FAQ.rst

```diff
@@ -0,0 +1,284 @@
+faster please use `its parallel capability <../examples/general/data_loading/parallel_external_source.html>`_.
+
+Q: Does DALI have any profiling capabilities?
+#############################################
+A: DALI doesn't have any built-in profiling capabilities, still it utilizes NVTX ranges
+and has a dedicated domain (so it is easy to find in the profile) to show its operations. So you can
+capture the profile using `NVIDIA Nsight Systems <https://developer.nvidia.com/nsight-systems>`_
+or any Deep Learning profile that also supports NVTX markers.
+
+Q: Does DALI support multi GPU/node training?
+#############################################
+A: Yes, DALI supports data-parallel and distributed data-parallel strategies (you can read more
+about these strategies `here <https://pytorch.org/tutorials/intermediate/ddp_tutorial.html#comparison-between-dataparallel-and-distributeddataparallel>`_).
+Its shards data into non-overlapping pieces using the number of shards (world size) and shard id (global rank), and
+uses device id to identify the GPU used in the particular node (local rank).
+
+More details can be also found it `this documentation section <../advanced_topics_sharding.html>`_
+
+Q: How to report an issue/RFE or get help with DALI usage?
+##########################################################
+A: DALI is an open-source project hosted on GitHub, you can ask questions and report issues
+using `this link <https://github.com/NVIDIA/DALI/issues>`_ directly.
+
+Q: Can DALI accelerate the loading of the data, not just processing?
+####################################################################
+A: DALI mostly focuses on processing acceleration, as in most cases the input data is compressed
+(audio, video, or images) and the input data is relatively small compared to the raw decoded output.
+Still, there are cases, where data is not compressed and loading it directly to the GPU is feasible.
+To support that case DALI can use `GPUDirect Storage <https://developer.nvidia.com/gpudirect-storage>`_
+inside Numpy GPU reader to bypass CPU and load the data directly to the GPU.
+
+Q: How can I obtain DALI?
+#######################################################
+A: DALI is available as a prebuilt python wheel binary -
+`see to learn how to install it <https://docs.nvidia.com/deeplearning/dali/user-guide/docs/installation.html>`_
+or as `a source code <https://github.com/NVIDIA/DALI>`_ that can be built on your own.
+
+Q: Which OS does DALI support?
+##############################
+A: DALI does support all major Linux distributions and indirectly Windows through
+`WSL <https://docs.nvidia.com/cuda/wsl-user-guide/index.html>`_. Regrettably, MacOS
+is not unsupported.
+
+Q: Where can I find the list of operations that DALI supports?
+##############################################################
+A: You can find a comprehensive list of operators available `here <../supported_ops.html>`_.
```

Comment:
Note: double check link in rendered docs

Reply:
Fixed